src/kibali/PermissionsImporter.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Kibali;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+
+namespace Kibali
+{
+    public class PermissionsImporter
+    {
+
+        public async Task<PermissionsDocument> Import(string inputFile, string permissionsFile)
+        {
+            var doc = new PermissionsDocument();
+            JsonElement rootObject;
+            JsonElement permissionsObject;
+
+            using (var client = new HttpClient())
```

Comment:
Maybe we should make this instance a static member?
https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines#recommended-use